h2o-core/src/main/java/water/rapids/ast/AstBuiltin.java

```diff
@@ -0,0 +1,8 @@
+package water.rapids.ast;
+
+/**
+ * (Replacement for AstPrimitive).
+ */
+public abstract class AstBuiltin<T extends AstBuiltin<T>> extends AstPrimitive<T> {
```

Comment:
This is the same kind of transition that we planned some time ago; only done in a more gradual fashion: the `AstPrimitive` and `AstBuiltin` names are synonyms and can actually coexist, and then the code could be moved from old name to new name in a gradual manner.